### DISCUSSION TITLE
Disabling Prisma Accelerate

### DISCUSSION THREAD

**Discussion by seejamescode** [prisma/prisma#24903] [Q&A]
### Question

👋 Is there documentation for disabling Accelerate? I may keep it, but my app only benefits from the connection pooling and not the cached queries. My assumption is that you must:

1. Build and deploy the app without `.$extends(withAccelerate())`
2. Change the database env to the direct connection
3. Disable Accelerate in the Prisma Accelerate dashboard's settings

**Comment by ludralph** [comment]
Hi @seejamescode 👋 

Thank you for raising this question.

Your assumption is correct.

Remove the withAccelerate extension from your Prisma Client setup. Here is an example of how you might have it set up:
```js
import { PrismaClient } from '@prisma/client'  
import { withAccelerate } from '@prisma/extension-accelerate'  

const prisma = new PrismaClient().$extends(withAccelerate())
```
You should modify it to:
```js
import { PrismaClient } from '@prisma/client'  

const prisma = new PrismaClient()
```
Change the database environment variable to the direct connection: Update your DATABASE_URL environment variable to use the direct connection string to your database. For example:
```
DATABASE_URL="postgres://username:password@host:port/db"
```
Disable Accelerate in the Prisma Accelerate dashboard's settings: You can disable Prisma Accelerate for the specified environment using the Prisma CLI. Here is the command to disable Accelerate:
```
npx prisma platform accelerate disable \
--environment $INSERT_ENVIRONMENT_ID \
--early-access
```
Make sure to replace `$INSERT_ENVIRONMENT_ID` with your actual environment ID. You can view your environment IDs with the [environment show](https://www.prisma.io/docs/platform/platform-cli/commands#environment-show) command.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
